### DISCUSSION TITLE
Postbox 7

### DISCUSSION THREAD

**Discussion by mokgit77** [Extended-Thunder/send-later#354] [Q&A] [answered]
This is a great ad on and was advertsed as working with Postbox in this article: 
https://lifehacker.com/the-best-add-ons-to-supercharge-postbox-1567358421

But when I try to load it I get "SendLater is not compatible with Postbox 7.0.48"

This is a sorely needed featuer in Postbox- any way to get is working in that version?
Thanks,
-Marc
![Screenshot 2021-07-16 19 13 37](https://user-images.githubusercontent.com/85697136/125991821-05eb28da-419e-4306-a7c7-86a69a637e9a.png)

**Comment by jph-sendlater** [comment, marked as answer]
Unfortunately, no. As far as I'm aware, Postbox is only allowing a very limited number of approved extensions. I believe they are also still using the (now legacy) format for extensions, so it wouldn't be compatible with the new version of Send Later anyway.

**Comment by mokgit77** [comment]
Ok thanks if this extension works with Thunderbird maybe I should switch to
that?

-Marc

Marc O'Krent
+44 141 846 1136  or
+1 702 867 1887
Fax: +44 20 7681 3787  or
         +1 702 924 4681
www.linkedin.com/mokla

**Reply by jph-sendlater** [reply]
I can't compare the two, since I've always used Thunderbird.

**Comment by mokgit77** [comment]
Ok- thanks.

On Fri, Jul 16, 2021 at 10:49 PM jph-sendlater ***@***.***>
wrote:

> I can't compare the two, since I've always used Thunderbird.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Extended-Thunder/send-later/discussions/354#discussioncomment-1014984>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUN2E4BRCIXXCIHYKEW2EVTTYCSMZANCNFSM5AQBHKUQ>
> .
>


-- 
Marc O'Krent
+44 141 846 1136 or
+1 702-867-1887
fax: +44 20 7681 3787 or
        +1 702 924 4681
www.linkedin.com/in/mokla

**Comment by jikamens** [comment]
I checked with the Postbox folks again about this recently, and unfortunately the version of Thunderbird that the current version of Postbox is based on is too old for us to be able to make Send Later compatible with it. I asked them to let me know if that changes and they want to work with me to make Send Later work in Postbox. Until I hear from them there's no way to move forward on this.
